packages/sdks/snippets/nextjs-sdk-next-14-app/.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
@midhunadarvin we can keep this folder name as `nextjs-sdk-next-app` till we add a new server for next 15 that uses our nextjs sdk